src/llvm.cpp

```diff
@@ -432,10 +683,11 @@ b32 CodegenLLVM(Package *p) {
     char *dir;
     char *name = GetFileName(p->path, &buff[0], &dir);
 
-    Debug _debug;
-    Debug *debug = NULL;
+    Debug debug;
     if (FlagDebug) {
         llvm::DIBuilder *builder = new llvm::DIBuilder(*module);
+
+        // TODO: Absolute path for dir
```

Comment:
This could be true, I know to get it to map to the right source file in Xcode I had to use an absolute address, I assume because my Working directory was not the one the compilation occurred from.

Reply:
I have checked what clang does and using absolute path would match its behavior
```llvm
!1 = !DIFile(filename: "ack.c", directory: "/Users/vdka/Source/temp")
```